File: StoryDomain/getStoryImgService/Controller/getStoryImgController.js
const get_story_img = async function (req, res) {
    var img = req.params['img'];
    fs.stat('./uploads/stories/' + img, function (err) {
        if (err) {
            res.status(200).send({ message: 'No se encontró la imagen' });
        } else {
            let path_img = './uploads/stories/' + img;
            res.status(200).sendFile(path.resolve(path_img));
        }
    });
}
module.exports = {
    get_story_img
}

File: StoryDomain/getStoryImgService/Controller/getStoryImgController.go
package controller

import (
	"log"
	"net/http"
	"os"
	"path/filepath"
	"github.com/gorilla/mux"
)


func GetStoryImg(w http.ResponseWriter, r *http.Request) {

	vars := mux.Vars(r)
	img, exists := vars["img"]
	if !exists {
		http.Error(w, "Image parameter not found", http.StatusBadRequest)
		return
	}

	imagePath := filepath.Join("uploads", "stories", img)

	_, err := os.Stat(imagePath)
	if os.IsNotExist(err) {
		http.Error(w, "Image not found", http.StatusNotFound)
		return
	} else if err != nil {
		log.Println("Error verifying image:", err)
		http.Error(w, "Internal Server Error", http.StatusInternalServerError)
		return
	}

	http.ServeFile(w, r, imagePath)
}